src/cluster_legacy.c

```diff
@@ -6234,3 +6234,16 @@ int handleDebugClusterCommand(client *c) {
 
     return 1;
 }
+
+int clusterAllowFailoverCmd(client *c) {
```

Comment:
Why is this per implementation? I assume we would always direct clients to the cluster failover command?

Reply:
Correct. It returns true in the flotilla implementation. It's just a simple way to keep the call site generic.